service/scaladsl/server/src/main/scala/com/lightbend/lagom/scaladsl/server/LagomApplicationLoader.scala

```diff
@@ -246,11 +247,14 @@ private[server] object ActorSystemProvider {
 
   val logger = Logger(classOf[LagomApplication])
 
+  case object ApplicationShutdownReason extends CoordinatedShutdown.Reason
```

Comment:
Should it use `ApplicationShutdownReason` from Play? I don't really understand the point of specifying a custom reason in the first place... does anything use it?

Reply:
We are not using it. Maybe we should simply use `UnknownReason` as default. Alternatively the one from Play, in case Play uses it.